modules/installation-configure-proxy.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-customizations.adoc
+// * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+// TODO: gcp
+
+// Questions:
+// - Can they provide multiple certs in additionalTrustBundle?
+// - Is it true that the trustedCA field is filled in? Slides say "trusted-ca-bundle",  but ConfigMap created is user-ca-bundle
+
+[id="installation-configure-proxy_{context}"]
+= Configuring the cluster-wide proxy during installation
+
+Production environments can deny direct access to the Internet and instead have
+an HTTP or HTTPS proxy available. You can configure a new {product-title}
+cluster to use a proxy by configuring the proxy settings in the
+`install-config.yaml` file.
+
+.Prerequisites
+
+* An existing `install-config.yaml` file.
+
+.Procedure
+
+. Edit your `install-config.yaml` file and add the proxy settings. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: my.domain.com
+proxy:
+  httpProxy: http://<username>:<pswd>@<ip>:<port> <1>
+  httpsProxy: https://<username>:<pswd>@<ip>:<port> <2>
+  noProxy: example.com <3>
+additionalTrustBundle: | <4>
+    -----BEGIN CERTIFICATE-----
+      <MY_TRUSTED_CA_CERT>
+    -----END CERTIFICATE-----
+...
+----
+<1> A proxy URL to use for creating HTTP connections outside the cluster. The
+URL scheme must be `http`.
+<2> A proxy URL to use for creating HTTPS connections outside the cluster. If
+this is not specified, then `httpProxy` is used for both HTTP and HTTPS
+connections. The URL scheme can either be `http` or `https`.
+<3> A comma-separated list of destination domain names, domains, IP addresses or
+other network CIDRs to exclude proxying. Preface a domain with `.` to include
+all subdomains of that domain. Use `*` to bypass proxy for all destinations.
+<4> If provided, the installer generates a ConfigMap named `user-ca-bundle` in
+the `openshift-config` namespace that contains the one or more additional CA
+certificates required for proxying HTTPS connections. The Cluster Network
+Operator then creates a `trusted-ca-bundle` ConfigMap merging these contents
+with the RHCOS trust bundle, and this ConfigMap is referenced in the Proxy
+object's `trustedCA` field. The `additionalTrustBundle` field is required unless
+the proxy's identity certificate is signed by an authority from the RHCOS trust
```

Comment:
You also only need this (for proxy connections) if you're using an `https` proxy URI.  No proxy cert to include if all your proxy connections are `http`.